packages/nameguard-js/src/nameguard-js.test.ts

```diff
@@ -4,6 +4,8 @@ import { mainnet, sepolia } from "viem/chains";
 import { createClient } from "./nameguard-js";
 
 const PROVIDER_URI_MAINNET = process.env.PROVIDER_URI_MAINNET;
+const PROVIDER_URI_SEPOLIA = process.env.PROVIDER_URI_SEPOLIA;
+const INVALID_NAMEGUARD_API_ENDPOINT = "http://localhost:1234";
```

Comment:
Thanks for your replies on why you're using this sometimes.

I understand the goal of proving that the secure primary name result had to have been computed locally and not via the NameGuard API.

To make this more clear, can you please add a comment above the `INVALID_NAMEGUARD_API_ENDPOINT` that explains why we're passing an invalid API endpoint sometimes. Ex: Prove that we calculate the secure primary name locally and not through the API.

Thanks